bulbs/content/models.py

```diff
@@ -597,7 +602,31 @@ def content_deleted(sender, instance=None, **kwargs):
         cls.search_objects.client.delete(index, doc_type, instance.id, ignore=[404])
 
 
+def delete_from_instant_article_api(sender, instance=None, **kwargs):
+    if getattr(settings, 'FACEBOOK_API_ENV', '').lower() == 'production':
+        fb_api_url = getattr(settings, 'FACEBOOK_API_BASE_URL', '')
+        fb_access_token = vault.read(settings.FACEBOOK_TOKEN_VAULT_PATH)
+
+        delete = requests.delete('{0}/{1}?access_token={2}'.format(
+            fb_api_url,
+            instance.instant_article_id,
+            fb_access_token
+        ))
+
+        status = delete.json()["success"]
```

Comment:
Also need to check `delete.ok` status before you access json(). On error code, `.json()` will raise.